config/locales/en.yml

```diff
@@ -10,6 +10,13 @@ en:
             question_text:
               blank: Enter a question
               too_long: Question text must be %{count} characters or less
```

Comment:
Do we still need these errors for the `page` model in here?

Reply:
Good spot, I think we should keep them there for the time being. It should be tidied up once we move wholesale to ActiveRecord.